go-manual/modules/ROOT/pages/query-simple.adoc

```diff
@@ -193,7 +193,7 @@ These are provided as an arbitrary number of callbacks from the 4th function arg
 
 It is recommended to *always specify the database explicitly* with the `neo4j.ExecuteQueryWithDatabase("<dbName>")` callback, even on single-database instances.
 This allows the driver to work more efficiently, as it does not have to resolve the home database first.
-If no database is given, the link:{neo4j-docs-base-uri}/operations-manual/current/manage-databases/introduction#manage-databases-default[user's home database] is used.
+If no database is given, the link:{neo4j-docs-base-uri}/operations-manual/current/database-administration/[user's home database] is used.
```

Comment:
This (and all later instances) should include the anchor to the home database section in that page.